### ISSUE TITLE
windows build

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#390]
if coala works on windows...
https://ci.appveyor.com/signup

**Comment by sils1297** [comment]
blocked by #196 in case that gets fixed, marking wontfix for now.

**Comment by sils1297** [comment]
first try on wip/sils/appveyor , works well so far.

**Comment by Makman2** [comment]
basic support is on master :)
